src/lib/types.ts

```diff
@@ -136,6 +136,39 @@ export interface TestData {
     required: ReqType;
 }
 
+
+/**
+ * (Internal) values for the test scores
+ */
+// eslint-disable-next-line no-shadow
```

Comment:
That's worrisome - what is being shadowed?

Reply:
I have searched all over the place in my code and there is no such 'Score' symbol anywhere. Worse: whatever symbol I use instead, the effects are the same, ie, this seems to be an eslint bug. And that is _really_ worrysome, because it casts a shadow on all eslint warnings...